pkg/proc/eval.go

```diff
@@ -680,7 +605,20 @@ func (scope *EvalScope) findGlobal(pkgName, varName string) (*Variable, error) {
 	if err != nil || v != nil {
 		return v, err
 	}
-	return nil, fmt.Errorf("could not find symbol value for %s.%s", pkgName, varName)
+	return nil, &errCouldNotFindSymbol{fmt.Sprintf("%s.%s", pkgName, varName)}
+}
+
+type errCouldNotFindSymbol struct {
+	name string
+}
+
+func (e *errCouldNotFindSymbol) Error() string {
+	return fmt.Sprintf("could not find symbol %s", e.name)
+}
+
+func isSymbolNotFound(e error) bool {
```

Comment:
Couldn't this just be `errors.As()`?

Reply:
No, but I can implement it with `errors.As`, which I have done (although after seeing the implementation of errors.As I'm really tempted to revert it).

Reply:
Yeah, it's not pretty, but if we ever wrap errors further this will be more correct.